src/test/assembly/panic-unwind-no-uwtable.rs

```diff
@@ -0,0 +1,10 @@
+// assembly-output: emit-asm
+// only-x86_64-unknown-linux-gnu
+// compile-flags: -C panic=unwind -C force-unwind-tables=n
+
+#![crate_type = "lib"]
+
+// CHECK: .cfi_startproc
```

Comment:
This needs a `CHECK-LABEL` as `panic!()` will cause rustc to generate a [fair number of functions](https://rust.godbolt.org/z/4Ph14Kcrx) into this CU.